lib/cisco_os_shim/nxapi/version.rb

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2015 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+module Cisco
+  module Shim
+    # Set version number for Cisco::Shim::NXAPI
+    module NXAPI
+      VERSION = '1.1.0-dev'
```

Comment:
I was thinking for this to be 1.1 since it's a successor to cisco_nxapi 1.0 - whereas the other two gems are 'new' and hence 1.0. But I'm not 100% on this.